lms/static/js/student_account/views/AccessView.js

```diff
@@ -161,9 +181,14 @@ var edx = edx || {};
                 this.loadForm( type );
             }
 
+            this.element.hide( $(this.el).find('.submission-success') );
             this.element.hide( $(this.el).find('.form-wrapper') );
             this.element.show( $form );
             this.element.scrollTop( $anchor );
+
+            // Update url without reloading page
+            History.pushState( null, document.title, '/account/' + type + '/' );
+            analytics.pageview( '/account/' + type );
```

Comment:
`pageview` is deprecated; [`page`](https://segment.com/docs/libraries/analytics.js/#page) should be used to record virtual pageviews. For example, [see here](https://github.com/edx/edx-platform/blob/487bac519ea3877e3dc5610a5097ec3457d66ae1/lms/static/js/verify_student/views/face_photo_step_view.js#L29).

When capturing virtual pageviews in the split verification and payment flow, we record a category in addition to a name. For the sake of consistency, I wonder if something like the following would be better:
```javascript
analytics.page( 'logistration', type );
```